engine/src/test/resources/testTopology.json

```diff
@@ -25,7 +25,7 @@
           "attributes": {
             "UNIFIER_LIMIT": 8,
             "STREAM_CODEC" : {
-              "com.datatorrent.common.codec.JsonStreamCodec" : {}
+              "com.datatorrent.stram.plan.logical.LogicalPlanConfigurationTest$TestStreamCodec" : {}
             }
```

Comment:
Yes, the test fails with the original JSON. So I changed the stream codec class in JSON.
It does make the change backword incompatible due to an additional validation. However, if attribute values are not serializable, the app currently fails during deployment. we need to catch attribute serializability in validate phase itself.